CordovaLib/CordovaLib.docc/upgrading-8.md

```diff
@@ -256,7 +256,7 @@ The following headers are deprecated due to adding global category extensions to
 * `<Cordova/NSMutableArray+QueueAdditions.h>`  
   This was only ever intended as an internal implementation detail.
 
-* `<Cordova/CDV.h>`
+* `<Cordova/CDV.h>`  
```

Comment:
Was a space added here?

Reply:
yes, this forces a newline in the documentation to match the other headers in this list: See https://apache.github.io/cordova-ios/documentation/cordova/upgrading-8?language=objc#Public-API-Removals for the current inconsistency